ESPixelStick/src/input/InputFPPRemotePlayFileFsm.cpp

```diff
@@ -224,14 +224,16 @@ void fsm_PlayFile_state_PlayingFile::Poll (uint8_t* Buffer, size_t BufferSize)
                 continue;
             }
 
+            uint32_t AdjustedFilePosition = FilePosition + CurrentSparseRange.DataOffset;
```

Comment:
Can be removed, we don't need dataoffset.

Reply:
I would need to remove the entire sparse range support (which is not a bad idea) in order to remove this. Since the fix I added was in the code that creates this number, then this code is correct as is.